src/fiskaltrust.ifPOS/v1/de/IDESSCD.cs

```diff
@@ -11,38 +11,92 @@ public interface IDESSCD
     {
         [OperationContract]
 #if WCF
-        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/starttransactionexportdata", Method = "POST")]
+        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/starttransaction", Method = "POST")]
 #endif
-        Task<StartTransactionResponse> StartTransactionExportDataAsync(StartTransactionRequest request);
+        Task<StartTransactionResponse> StartTransactionAsync(StartTransactionRequest request);
 
         [OperationContract]
 #if WCF
-        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/updatetransactionexportdata")]
+        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/updatetransaction", Method = "POST")]
 #endif
-        Task<UpdateTransactionResponse> UpdateTransactionExportDataAsync(UpdateTransactionRequest request);
+        Task<UpdateTransactionResponse> UpdateTransactionAsync(UpdateTransactionRequest request);
 
         [OperationContract]
 #if WCF
-        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/finishtransactionexportdata")]
+        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/finishtransaction", Method = "POST")]
 #endif
-        Task<FinishTransactionResponse> FinishTransactionExportDataAsync(FinishTransactionRequest request);
+        Task<FinishTransactionResponse> FinishTransactionAsync(FinishTransactionRequest request);
+
 
         [OperationContract]
 #if WCF
         [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/tseinfo", Method = "GET")]
 #endif
         Task<TseInfo> GetTseInfoAsync();
 
+
         [OperationContract]
 #if WCF
-        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/exportdata", Method = "GET")]
+        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/tsestate", Method = "POST")]
 #endif
-        Task<TseExportDataResult> ExportDataAsync();
+        Task<TseState> SetTseStateAsync(TseState state);
+
 
         [OperationContract]
 #if WCF
-        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/tsestate", Method = "POST")]
+        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/registerclientid", Method = "POST")]
 #endif
-        Task<TseState> SetTseStateAsync(TseState state);
+        Task<ClientIdResponse> RegisterClientId(ClientIdRequest request);
+
+        [OperationContract]
+#if WCF
+        [WebInvoke(BodyStyle = WebMessageBodyStyle.Bare, UriTemplate = "v1/unregisterclientid", Method = "POST")]
+#endif
+        Task<ClientIdResponse> UnRegisterClientId(ClientIdRequest request);
```

Comment:
I think it could make sense to differentiate between `Regsiter` and `Unregister` here from a object perspective to have separate messages. `RegisterClientRequest` and `UnregisterClientRequest`

Reply:
If we need to make changes in the future, that could be helpful to be at lease less-breaking.